sycl/source/detail/usm/usm_impl.cpp

```diff
@@ -19,98 +19,98 @@ using alloc = cl::sycl::usm::alloc;
 namespace detail {
 namespace usm {
 
-void *alignedAlloc(size_t alignment, size_t size, const context *ctxt,
-                   const device *dev, alloc kind) {
-  cl_int error;
-  cl_context c =
-      pi::cast<cl_context>(detail::getSyclObjImpl(*ctxt)->getHandleRef());
-  cl_device_id id;
-
-  void *retVal = nullptr;
-
-  switch (kind) {
+void *alignedAlloc(size_t Alignment, size_t Size, const context &Ctxt,
+                   const device *Dev, alloc Kind) {
```

Comment:
Minor. Could you please pass "Dev" be reference?

Reply:
If I do, I'll have to add a 2nd version that doesn't take Dev and logic elsewhere to use the right version.  hostAllocations don't have a Device, so it's nice for this to be able to be null.  You can't do that with a Reference, right?